compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.h

```diff
@@ -59,6 +59,10 @@ std::unique_ptr<Pass> createAMDAIEControlCodeLoopUnrollPass();
 /// Pass to create a single AIE workgroup.
 std::unique_ptr<Pass> createAMDAIECreateAIEWorkgroupPass();
 
+/// Pass to create logical objectfifo link operations, explicitly linking input
```

Comment:
Nit: linking inputs and outputs. Same for the summary in Passes.td.

Reply:
Thanks for the catch.